src/datasources/postgres/EquipmentDataSource.ts

```diff
@@ -63,14 +71,49 @@ export default class PostgresEquipmentDataSource
       .select('*')
       .orderBy('name', 'asc')
       .modify(qb => {
-        if (equipmentIds) {
+        if (equipmentIds && equipmentIds[0] !== 0) {
```

Comment:
Why is it 0 check here?

Reply:
I will try to fix this. It is used to load all equipments but I will try to pass an emptpy array and make it work without this check. Thanks for the good point.